post/templates/post/add_comment.html

```diff
@@ -0,0 +1,25 @@
+{% extends 'post/base.html' %}
+{% block title %} New Comment {% endblock %}
+{% block content %}
+   
+ <h1>Add Comment</h1>
+ <div class="form-group">
+    <form method="POST">
+        {% csrf_token %}
+   {% if user.is_authenticated %}
+        {{ form.media }}
```

Comment:
Hey, are we planning to upload anything with comment, because I don't know why we have here form.media.

Reply:
I think we should do it first... later on we can add things , and make changes

Reply:
i will also ask the client first , don't need to do everything now and need to change later

Reply:
It looks confusing for me because it looks we are uploading file when we are not.
We can always add this line when we decide to add this feature:) I think the code looks cleaner this way.